colossalai/fx/passes/meta_info_prop.py

```diff
@@ -232,7 +233,7 @@ def output(self, target: 'Target', args: Tuple[Argument, ...], kwargs: Dict[str,
             flop_count (Tuple): The flop count for (fwd_flop, bwd_flop).
             mem_stat (Tuple): The memory statistics for (fwd_tmp, fwd_out, bwd_tmp, bwd_out)
         """
-        return args[0], (0, 0), (0, 0, 0, 0)
+        return args[0], {'fwd_in': activation_size(args[0])}
```

Comment:
Do not return data as dict, return it as a dataclass object so that it has clearer semantics.